app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala

```diff
@@ -138,7 +141,8 @@ class SchemeMembersListController @Inject()(
                 optCurrentVersion = request.currentVersion,
                 optPreviousVersion = request.previousVersion,
                 compilationOrSubmissionDate = request.userAnswers.get(CompilationOrSubmissionDatePage(srn)),
-                showBackLink = showBackLink
+                showBackLink = showBackLink,
+                prePopNotChecked = request.userAnswers.get(MembersDetailsChecked(srn)).contains(false)
```

Comment:
It might be an overkill, but potentially you can utilise the existing `isPrePopulation` method as an additional guard condition here, e.g. something like:
`` prePopNotChecked = isPrePopulation && request.userAnswers.get(MembersDetailsChecked(srn)).contains(false) ``
to make sure this applies for pre-population only. Even if in theory for not pre-populated records this should never be false, it will add some additional certainty IMO.